components/cam/bld/configure

```diff
@@ -2073,11 +1873,6 @@ if ($clubb_sgs == 1) {
     $cfg_cppdefs .= " -DCLUBB_REAL_TYPE=dp";
 }
 
-# SHOC
```

Comment:
just noting locations where shoc/p3 might be removed.

Reply:
retain all SHOC definitions